src/main/java/fruitshop/db/Storage.java

```diff
@@ -0,0 +1,24 @@
+package fruitshop.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
According to the encapsulation principle, the 'storage' field should be private. Exposing it even through a getter method can be risky as it allows direct access to the internal representation. Consider returning a copy of the storage map or an unmodifiable view.